website/snippets/_legacy-sl-callout.md

```diff
@@ -0,0 +1,11 @@
+:::tip Upgrade to access the new dbt Semantic Layer
```

Comment:
I suggest using "important" It's the same color but a little more pressing than the lightbulb icon. We should come up with a philosophy for notes. 
```suggestion
:::important Upgrade to access the new dbt Semantic Layer
```

Reply:
ah good one! thank you!